resources/views/discord/new.blade.php

```diff
@@ -45,7 +45,7 @@
                                 </a>
                             @else
                                 <a href="#" style="text-decoration: none;">
-                                    <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em> &thinsp; Discord Account {{ $_account->discord_id }} Linked</button>
+                                    <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em> &thinsp; Discord Account {{ $_account->discord_user->username ?? $_account->discord_id }}#{{ $_account->discord_user->discriminator ?? ' ' }} Linked</button>
```

Comment:
A check on the `discriminator` would likely be more appropriate as the presence of a `username` attribute has no bearing on the output.

Reply:
Is this ok? https://github.com/VATSIM-UK/core/pull/2029/commits/b1e8d21b6044e905acc4d895361d1ff57f61a7bf